dlp/snippets/inspect/inspect_test.go

```diff
@@ -681,3 +688,207 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
 		t.Errorf("TestInspectTableWithCustomHotword got %q, want %q", got, want)
 	}
 }
+
+func TestInspectDataStoreSendToScc(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+	u := uuid.New().String()[:8]
+	datastoreNamespace := fmt.Sprint("golang-samples" + u)
+	datastoreKind := "task"
+
+	if err := inspectDataStoreSendToScc(&buf, tc.ProjectID, datastoreNamespace, datastoreKind); err != nil {
+		t.Fatal(err)
+	}
+
+	got := buf.String()
+	if want := "Job created successfully:"; !strings.Contains(got, want) {
+		t.Errorf("InspectBigQuerySendToScc got %q, want %q", got, want)
+	}
+}
+
+func TestInspectWithStoredInfotype(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer client.Close()
+	outputBucket, err := testutil.CreateTestBucket(ctx, t, client, tc.ProjectID, bucket_prefix)
```

Comment:
issue: move test setup & teardown into a `TestMain(m *testing.M)` function.

Reply:
As discussed in this https://github.com/GoogleCloudPlatform/golang-samples/pull/3222#discussion_r1312033185, The available functions in `testutil` are not capable of implementing the setup from `TestMain(m *testing.M)` but still I have tried my best to teardown in `TestMain` function.